packages/react/src/elements/components/Typography.js

```diff
@@ -2,77 +2,208 @@ import React from "react";
+    <Typography
+      type="text"
+      bold={bold}
+      size={size}
+      color={color}
+      disabled={disabled}
+      text={children}
+    />
+  );
 }
 
-function Sub2Component(props) {
-  return <Typography type="sub2" text={props.children} />;
+export function Sub1({ children, size, bold, color, disabled }) {
+  console.warn(
+    "Component Sub1 is deprecated and will be removed in the next version"
+  );
+  return (
+    <Typography
+      type="sub1"
+      bold={bold}
+      size={size}
+      color={color}
+      disabled={disabled}
+      text={children}
+    />
+  );
 }
 
-function BodyComponent(props) {
-  return <Typography type="body" text={props.children} />;
+export function Sub2({ children, size, bold, color, disabled }) {
+  console.warn(
+    "Component Sub2 is deprecated and will be removed in the next version"
+  );
+  return (
+    <Typography
+      type="sub2"
+      bold={bold}
+      size={size}
+      color={color}
+      disabled={disabled}
+      text={children}
+    />
+  );
 }
 
-function BoldComponent(props) {
-  return <Typography type="bold" text={props.children} />;
+export function Body({ children, size, bold, color, disabled }) {
+  console.warn(
+    "Component Body is deprecated and will be removed in the next version"
+  );
+  return (
+    <Typography
+      type="body"
+      bold={bold}
+      size={size}
+      color={color}
+      disabled={disabled}
+      text={children}
+    />
+  );
 }
 
-function DisabledComponent(props) {
-  return <Typography type="disabled" text={props.children} />;
+export function Bold({ children, size, bold, color, disabled }) {
+  console.warn(
+    "Component Bold is deprecated and will be removed in the next version"
+  );
+  return (
+    <Typography
+      type="bold"
+      bold={bold}
+      size={size}
+      color={color}
+      disabled={disabled}
+      text={children}
+    />
+  );
 }
 
-function CaptionComponent(props) {
-  return <Typography type="caption" text={props.children} />;
+export function Disabled({ children, size, bold, color, disabled }) {
+  console.warn(
+    "Component Disabled is deprecated and will be removed in the next version"
+  );
+  return (
+    <Typography
+      type="disabled"
+      bold={bold}
+      size={size}
+      color={color}
+      disabled={disabled}
+      text={children}
+    />
+  );
+}
+
+export function Caption({ children, size, bold, color, disabled }) {
+  console.warn(
+    "Component Caption is deprecated and will be removed in the next version"
+  );
+  return (
+    <Typography
+      type="caption"
+      bold={bold}
+      size={size}
+      color={color}
+      disabled={disabled}
+      text={children}
+    />
+  );
 }
 
 const docgenInfo = {
   props: {
+    bold: {
+      description: "Whether to render bold text"
+    },
+    color: {
+      description:
+        "A text color. One of: 'hig-white', 'hig-cool-gray-70', 'hig-blue-50', 'hig-green-good', 'hig-yellow-warning', 'hig-red-alert'"
```

Comment:
It would be cleaner to pull this array of color values from HIGVanilla Typography.AvailableColors. We haven't been doing that for the descriptions, but something to think about.

Reply:
:+1:. Initially I thought I'd have to globally export these alongside the Typography component, but of course they can be static variables.